mautic/mautic
# Update grapesjs to 0.21.12

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴🟢
| Deprecations?                          | 🔴🟢
| BC breaks? (use the c.x branch)        | 🔴🟢
| Automated tests included?              | 🔴🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This updates the version of GrapesJS to the latest bugrelease. This is to keep the Builder up-to-date.

Changelogs:
- https://github.com/GrapesJS/grapesjs/releases/tag/0.21.11
- https://github.com/GrapesJS/grapesjs/releases/tag/v0.21.12







---
### 📋 Steps to test this PR:



These are minor changes so I don't expect issues. Best way to test is simply to make an Email & a Landing page, make changes and save + reopen to see if it all behaves as expected.

For example:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make a new contact and new segment
3. Start a new mail and open the Builder
4. Choose one of the existing themes
5. Add various blocks eg. Text block, link, …
6. Make content changes to them
7. Save, close and open -> are content and positions correct?
8. Reorder the blocks, save, close and reopen -> are content and positions correct?
